python/lsst/obs/subaru/strayLight/rotatorAngle.py

```diff
@@ -248,9 +249,14 @@ def inrStartEnd(header):
         Instrument rotator angle at end of exposure, degrees.
     """
 
-    inst_pa = header.getDouble('INST-PA')
-    ra_t_d = header.getDouble('CRVAL1')
-    de_t_d = header.getDouble('CRVAL2')
+    visitInfo = exposure.getInfo().getVisitInfo()
+    header = exposure.getMetadata()
+
+    inst_pa = 270.0 - visitInfo.getBoresightRotAngle().asAngularUnits(lsst.geom.degrees)
+    ra_t_sp, de_t_sp = visitInfo.getBoresightRaDec()
```

Comment:
I had not idea this would work. Now I need to find out how `getBoresightRaDec()` can return a `SpherePoint` object or ra and dec separately.

Reply:
I also wasn't sure it would work, but it looked likely based on the results I had in the debugger.